explore-analyze/discover/discover-get-started.md

```diff
@@ -6,21 +6,18 @@ applies_to:
   serverless: ga
 products:
   - id: kibana
+description: Step-by-step tutorial for exploring data with Discover by selecting data views, filtering documents, analyzing fields, and creating visualizations using sample or your own data.
 ---
 
 # Explore fields and data with Discover [discover-get-started]
 
-Learn how to use **Discover** to:
+**Discover** provides a flexible interface for exploring your {{product.elasticsearch}} data. Learn how to select and filter data, explore field structures and document content, and present your findings in visualizations. This tutorial walks you through the core workflows for analyzing data in **Discover**, from loading data to creating and saving visualizations.
```

Comment:
`Learn how to do x` followed by `This tutorial walks you through y` reads a little goofy to me. Can we combine those sentences? Could even steal from the meta description.